benchmarks/riccati.jl

```diff
@@ -3,12 +3,13 @@
 module riccati
 
 using LinearAlgebra
+using MKL
 
 # Riccat recursion
 
 function riccati_trf(N, nx, nu, BAt, RSQ, L, LN, BAtL, M)
 
-	LN = copy(RSQ[nu+1:nu+nx, nu+1:nu+nx]);
+	LN = RSQ[nu+1:nu+nx, nu+1:nu+nx];
```

Comment:
Maybe remove the semicolon(s) here, too, while you are at it?
```suggestion
	LN = RSQ[nu+1:nu+nx, nu+1:nu+nx]
```